domain/src/main/java/org/oppia/domain/util/InteractionObjectExtensions.kt

```diff
@@ -60,6 +63,10 @@ private fun StringList.toAnswerString(): String {
   return htmlList.joinToString()
 }
 
+private fun ListOfSetsOfHtmlStrings.toAnswerString(): String {
+  return setOfHtmlStringsList.joinToString(prefix = "[",postfix = "]") { it.htmlList.joinToString(" ") }
```

Comment:
Just to make sure I am reading this correct. This would basically do:
`[a b c], [1 2], [w x y z]`
for 
```{
  setOfHtmlStrings: [
    ["a", "b", "c"],
    ["1", "2"],
    ["w", "x", "y", "z"]
  ]
}
```
right? 
Super tiny nit but can we keep the delimiter as "," instead of " " for it.htmlList? 
So it would be  `[a,b,c], [1,2], [w,x,y,z]`

Reply:
Output for the above [a b c, 1 2, w x y z]

Reply:
Oops sorry I misread it! Yes you are right. How about we make it as 
`[a,b,c], [1,2], [w,x,y,z]`
That feels a lot easier to read

Reply:
done